lib/agent/check.js

```diff
@@ -100,6 +100,7 @@ function getResponse(uri, req, cb)
     });
   }
 
+  // OCSP is performed without HTTPS
```

Comment:
What do you mean by `without HTTPS`? Maybe `insecure HTTP` is better? And why do we need this comment here?

Reply:
Removed comment